config.h.meson

```diff
@@ -0,0 +1,98 @@
+/* config.h.in.  Generated from configure.ac by autoheader.  */
+
+/* Define to 1 if you have the <inttypes.h> header file. */
+#mesondefine HAVE_INTTYPES_H
+
+/* Define to 1 if you have the <memory.h> header file. */
+#mesondefine HAVE_MEMORY_H
+
+/* Define if SELinux is available */
+#mesondefine HAVE_SELINUX
```

Comment:
Specifically having things like this seems pretty broken.  It should be detected at build time, no?